Tests/UnitTests/EdgeHitTests.swift

```diff
@@ -79,8 +80,102 @@ class EdgeHitTests: XCTestCase {
+        let streamingSettings = Streaming(recordSeparator: "A", lineFeed: "B")
+        let edgeHit = ExperienceEventsEdgeHit(endpoint: INTERACT_ENDPOINT_PROD,
+                                              datastreamId: CONFIG_ID,
+                                              request: EdgeRequest(meta: RequestMetadata(konductorConfig: KonductorConfig(streaming: streamingSettings), state: nil, sdkConfig: SDKConfig(datastream: Datastream(original: "OriginalDatastreamID")), configOverrides: nil), xdm: nil, events: [["test": "data"]]))
+
+        XCTAssertTrue(expectedPayload == payloadToDict(payload: edgeHit.getPayload()))
+    }
+
+    func testExperienceEventsEdgeHit_getPayload_withDatastreamConfigOverride() {
+        let configOverrides: [String: Any] = [
+            "com_adobe_experience_platform": [
+              "datasets": [
+                "event": [
+                  "datasetId": "testEventDatasetIdOverride"
+                ],
+                "profile": [
+                  "datasetId": "testProfileDatasetIdOverride"
+                ]
+              ]
+            ],
+            "com_adobe_analytics": [
+              "reportSuites": [
+                "rsid1",
+                "rsid2",
+                "rsid3"
+                ]
+            ],
+            "com_adobe_identity": [
+              "idSyncContainerId": "1234567"
+            ],
+            "com_adobe_target": [
+              "propertyToken": "testPropertyToken"
+            ]
+          ]
+
+        let json = """
+                {"meta" :
+                    {"konductorConfig" :
+                        {"streaming" :
+                            {"enabled" : true,"recordSeparator" : "A","lineFeed" : "B"}
+                        },
+                        "configOverrides" : {
+                            "com_adobe_experience_platform": {
+                              "datasets": {
+                                "event": {
+                                  "datasetId": "testEventDatasetIdOverride"
+                                },
+                                "profile": {
+                                  "datasetId": "testProfileDatasetIdOverride"
+                                }
+                              }
+                            },
+                            "com_adobe_analytics": {
+                              "reportSuites": [
+                                "rsid1",
+                                "rsid2",
+                                "rsid3"
+                                ]
+                            },
+                            "com_adobe_identity": {
+                              "idSyncContainerId": "1234567"
+                            },
+                            "com_adobe_target": {
+                              "propertyToken": "testPropertyToken"
+                            }
+                          }
```

Comment:
nit: fix indentation so it's clear this section is under `meta` and not `konductorConfig`.